wallet/src/de/schildbach/wallet/ui/MainActivity.kt

```diff
@@ -102,6 +115,21 @@ class MainActivity : AbstractBindServiceActivity(), ActivityCompat.OnRequestPerm
         setupBottomNavigation()
     }
 
+    fun initViewModel() {
+        viewModel = ViewModelProvider(this)[MainActivityViewModel::class.java]
+        viewModel.blockchainStateData.observe(this, Observer {
+            // just to trigger data loading
+        })
+        viewModel.isPlatformAvailableData.observe(this, Observer {
+            // just to trigger data loading
+        })
```

Comment:
Those empty observers could be replaced by other way of triggering data loading but let's leave it as it is
I'll fix it when working on "JoinDash" shortcut when https://github.com/dashevo/dash-wallet/pull/516 in merged (it will be merged after this PR)